tangelo/toolboxes/ansatz_generator/tests/test_hea.py

```diff
@@ -104,6 +106,66 @@ def test_hea_H4_doublecation(self):
         energy = sim.get_expectation_value(qubit_hamiltonian, hea_ansatz.circuit)
         self.assertAlmostEqual(energy, -0.795317, delta=1e-4)
 
+    def test_hea_circuit_variational_reference_state(self):
+        """ Verify variational gate parameters are frozen in a reference state circuit"""
+
+        # Create simple reference circuit:
+        ref_circuit = Circuit([
+            Gate('Ry', 0, parameter=1.0, is_variational=True),
+            Gate('Ry', 1, parameter=1.0, is_variational=True),
+            Gate('CX', 0, 1),
+        ])
+
+        # create HEA ansatz with reference circuit
```

Comment:
Could we start all these comments with a capital letter for consistency please ?

Reply:
Fixed in [commit 37a9497](https://github.com/goodchemistryco/Tangelo/pull/398/commits/37a94975fdc86946aeb7c4a7e5a0f9a7555aa9b7).